scripts/build-deb.py

```diff
@@ -59,8 +59,24 @@
         )
     print("✅ Checksum of original source package matched.")
 
-    # Unpack the source package
-    subprocess.run(['dpkg-source', '-x', dsc_file], cwd=temp_dir, check=True)
+    script = config.get('script')
+    if script:
+        if not os.path.isabs(script):
+            config_dir = os.path.dirname(args.config)
+            script = os.path.abspath(os.path.join(config_dir, script))
+
+        env = os.environ.copy()
+        env['DSC_FILE'] = dsc_file
```

Comment:
I'd prefer if we passed the whole yaml file to the script, but I know it's not fun to parse yaml from shell; that's actual actually why I wrote build-deb in python, but now we're extending it to call a shell script :)

I guess this is a temporary solution and we need to get rid of this debdiff juggling logic ASAP, but this will do for now.